lib/deps.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+function makeErrorModule(error) {
+  return new Proxy(
+    {},
+    {
+      get: () => {
+        throw error;
+      },
+      set: () => {
+        throw error;
+      }
+    }
+  );
+}
+
+function retrieveKerberos() {
+  try {
+    return require('kerberos');
+  } catch (err) {
+    const noKerberosError = new Error(
+      'The `kerberos` module was not found. Please install it and try again.'
+    );
+    return makeErrorModule(noKerberosError);
+  }
+}
+
+// Facilitate loading EJSON optionally
+function retrieveEJSON() {
```

Comment:
this can go away since Extended JSON support is in the `bson` module

Reply:
Yep and _I think_ that there's no imports for EJSON in the driver only in the tests, will drop